README.md

```diff
@@ -62,6 +62,11 @@ Use the Internet Identity canister in your local dfx project by adding the follo
   }
 }
 ```
+To deploy, run `dfx deploy`.
+
+To access Internet Identity use one of the following URLs:
```

Comment:
Maybe following to express the fact that it's also the URL to configure for auth-client?

To access Internet Identity or configure it for your dapp, you can use one of the following URLs: